backend.native/tests/build.gradle

```diff
@@ -4309,7 +4311,7 @@ dynamicTest("interop_cleaners_second_thread") {
 }
 
 dynamicTest("interop_cleaners_leak") {
-    disabled = (project.testTarget != null && project.testTarget != project.hostName)
+    disabled = (project.target.name != project.hostName)
```

Comment:
I merged in another two tests that use `std::thread` here: https://github.com/JetBrains/kotlin-native/pull/4494/files#diff-d3c79e0be6518d77481d6b73a22db4f79ab351158fda879bc88bb6b237fd6512R4330 I checked that the tests pass fine with `project.target.name != project.hostName` on everything but mingw.